content/en/dynamic_instrumentation/enabling/nodejs.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Enable Dynamic Instrumentation for Node.js
+aliases:
+    - /tracing/dynamic_instrumentation/enabling/nodejs/
+private: false
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+    - link: 'agent'
+      tag: 'Documentation'
+      text: 'Getting Started with Datadog Agent'
+---
+
+{{< beta-callout-private url="https://forms.gle/TODO" >}}
+    Dynamic Instrumentation for Node.js is in limited preview, and is not available to all customers.
+    Request access to join the waiting list.
+    Note that <a href="#limitations">some limitations</a> apply to the preview.
+{{< /beta-callout-private >}}
+
+Dynamic Instrumentation is a feature of supporting Datadog tracing libraries. If you are already using [APM to collect traces][1] for your application, ensure your Agent and tracing library are on the required version. Then, go directly to enabling Dynamic Instrumentation in step 4.
+
+## Installation
+
+1. Install or upgrade your Agent to version [7.45.0][6] or higher.
+2. If you don't already have APM enabled, in your Agent configuration, set the `DD_APM_ENABLED` environment variable to `true` and listening to the port `8126/TCP`.
```

Comment:
is that required for DI?

Reply:
See related internal discussion on https://dd.slack.com/archives/G01AQ0HN9C3/p1737026330302969

I was also a bit surprised to see this, because I think the vast majority of customers won't need to do this grunt work (they only would if they had previously _explicitly_ opted out of APM by setting this to false), but hopefully most will just read the "If you don't already have APM enabled" part and move on.